specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -496,24 +653,32 @@ model ResourceMetric {
   resourceType: string;
 }
 
-@doc("Load test run model")
-@resource("test-runs")
-model TestRuns {
-  // FIXME: is testRunId optional or required in the response? Resource key and path params cannot be optional
-  @key
-  @path
-  @doc("Unique test run name as identifier")
-  @visibility("read")
-  testRunId: string;
-}
+// @doc("Load test run model")
+// @resource("test-runs")
+// model TestRuns {
+//   // FIXME: is testRunId optional or required in the response? Resource key and path params cannot be optional
+//   @key
+//   @path
+//   @doc("Unique test run name as identifier")
+//   @visibility("read")
+//   testRunId: string;
+// }
 
 @doc("Load test run model")
 model TestRun {
-  ...TestRuns,
+  @doc("Unique test run name as identifier")
+  @visibility("read")
+  @maxLength(50)
+  @minLength(2)
+  testRunId?: string;
 
   @doc("Pass fail criteria for a test.")
   passFailCriteria?: PassFailCriteria;
 
+  @doc("Auto stop criteria for a test.")
+  @added(APIVersions.v2023_04_01_preview)
+  autoStopEnabled?: AutoStopCriteria;
```

Comment:
Should this be a list to support one or more criteria in the future?

Reply:
Can we update the specification even though we currently don't have support for this. Wouldn't it be misleading to users?

Reply:
It wouldnt be I assume. We have to check the doc on what is the guidance

Reply:
Yeah sure. I was suggesting that it might be misleading since it supports a list of values in spec, but we won't be accepting list in API.